src/main/webapp/script.js

```diff
@@ -20,16 +20,28 @@ function getUser() {
   fetch('/users').then(response => response.json());
 }
 
+function prepareSearch() {
+  document.getElementById("start-date").min = new Date().toISOString().slice(0, 10);
+  document.getElementById("end-date").min = new Date().toISOString().slice(0, 10);
```

Comment:
Why do you need to slice it?

Reply:
I wasn't sure but it was done this way somewhere else in the code.